### DISCUSSION TITLE
Why does entering '@Tasklist' in a page cause it to appear in the "Waiting" section of the tasklist?

### DISCUSSION THREAD

**Discussion by tscahill** [zim-desktop-wiki/zim-desktop-wiki#2678] [Q&A]
I'm getting back to Zim after a prolonged absence. I have been unwell. I admit up front that I remain under somewhat of a brain fog through this long recovery, so it wouldn't surprise me if I've overlooked something obvious.

Today, resolved to be "really productive", I conquered my fear _and clicked on the 'Task List' button at the top of Zim_. I was steeled for the resulting sight: a lot of waiting ToDos. Hoping to whittle the list down to something less intimidating, _I started clicking on the various filters on the left._ 

_Clicking on "Active" filter_ had the soothing effect of vanishing some of the tasks. For fun, _I clicked on the 'Waiting' filter,_ knowing I had not got around to using that feature. To my surprise, _a task remained_. So much for productivity. 

It _was_ actually a pending task, to quite literally come back to the functions of the Task List plugin to get more utility out of it. Previously, I had created the tag @Tasklist specifically on that page, to help me get back to it someday. This morning, after an hour or so, I have isolated it as the cause; to be what it is about the text of that page that causes it to be returned by the 'Waiting' filter. I am appending the full test of the related page below.

[Tasklist.txt](https://github.com/user-attachments/files/17013278/Tasklist.txt)

At first I was confident that is was the actual text "Waiting", that was the cause. Nope. I checked the properties of the Task List plugin, and made sure there were no entries in either of the text boxes for labels or tags for "waiting" tasks. I made a new page and opened its source file and pasted the full contents of 'Tasklist.txt' into it. I deleted the word waiting, and saved the page. I refreshed the zim index and re-activated the task list. The copied page joined the first one in appearing in the waiting filter list. 

Through a process of elimination, I successively removed and added various portions of the copied text to determine what was matching the filter. I changed the name of the page twice. @Tasklist is the culprit. 

Is this intentional? Kudos, if so. Nice _undocumented_ feature?

I have several other plugins enabled.

**Comment by jaap-karssenberg** [comment]
You can configure tags that trigger the waiting behavior. Depending on the
version of zim that can be a hidden config field. Maybe you did configure
that before and the config file is still on your system?

If you don't see the config on the plugin properties dialoog,
Have a look at the file with notebook properties (notebook.zim).

How this helps

Jaap

Op ma 16 sep. 2024 15:12 schreef tscahill ***@***.***>:
